lib/src/core/splash/list_tile_splash.dart

```diff
@@ -171,27 +177,27 @@ class NFListTileInkRipple extends InteractiveInkFeature {
   final RectCallback? _clipCallback;
   final TextDirection _textDirection;
 
-
   bool isCancelled = false;
 
   late Animation<double> _radius;
   late AnimationController _radiusController;
 
-  late Animation<int> _fadeIn;
+  late Animation<double> _fadeIn;
   late AnimationController _fadeInController;
 
   // Animation<int> _fadeOut;
   late AnimationController _fadeOutController;
 
   /// Used to specify this type of ink splash for an [InkWell], [InkResponse]
   /// or material [Theme].
-  static const InteractiveInkFeatureFactory splashFactory = _NFListTileInkRippleFactory();
```

Comment:
It looks like these files are not correctly formatted, whether I format them with 80 or 120 line length, I always get some changes. I suggest to format all files after this PR is merged.